### ISSUE TITLE
TypeScript error in editor when configuring Drizzle plugin in `knip.config.ts`

### ISSUE THREAD

**Issue by jjjjonathan** [webpro-nl/knip#829]
### Prerequisites

- [X] I'm using the latest version
- [X] I've read the relevant [documentation](https://knip.dev)
- [X] I've searched for [existing issues](https://github.com/webpro-nl/knip/issues?q=is%3Aissue)
- [X] I've checked the [list of known issues](https://knip.dev/reference/known-issues)
- [X] I've read the [issue reproduction guide](https://knip.dev/guides/issue-reproduction)

### Reproduction url

https://stackblitz.com/edit/github-hyliy9?file=knip.config.ts

### Reproduction access

- [X] I've made sure the reproduction is publicly accessible

### Description of the issue

When using a typed config file, the type for the Drizzle plugin does not seem to exist:
```
Object literal may only specify known properties, and 'drizzle' does not exist in type '{ exclude?: ("dependencies" | "exports" | "files" | "devDependencies" | "optionalPeerDependencies" | "unlisted" | "binaries" | "unresolved" | "types" | "nsExports" | "nsTypes" | "duplicates" | "enumMembers" | "classMembers")[] | undefined; ... 98 more ...; workspaces?: Record<...> | undefined; }'.ts(2353)
```
The plugin itself works fine, though, I only noticed an issue with the types.

**Comment by webpro** [comment]
:rocket: _This issue has been resolved in v5.36.6. See [Release 5.36.6](https://github.com/webpro-nl/knip/releases/tag/5.36.6) for release notes._

_Using Knip in a commercial project? Please consider [becoming a sponsor](https://knip.dev/sponsors)._

**Comment by jjjjonathan** [comment]
@webpro thanks for resolving so quickly!
